#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_change_pkgIndex.in.dpatch by  <muammar@localhost>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Changing pkgIndex.in

@DPATCH@

--- tcltls-1.5.0/pkgIndex.tcl.in	2003-12-15 14:46:20.000000000 -0400
+++ pkgIndex.tcl.in	2007-04-11 13:26:57.000000000 -0400
@@ -5,5 +5,5 @@
 #    replace the original which didn't include the commands from "tls.tcl".
 #
 
-package ifneeded tls 1.5 "[list load [file join $dir @RELPATH@ @tls_LIB_FILE@] ] ; [list source [file join $dir tls.tcl] ]"
+package ifneeded tls 1.50 "[list load [file join $dir .. libtls1.50.so] ] ; [list source [file join $dir tls.tcl] ]"
 
